src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -157,7 +157,7 @@
 			//Observer Enrollments.
 			put("ObserverEnrollment", 0);
 			put("StudentEnrollment", 0);
-			put("TaEnrollment", 1);
+			put("TAEnrollment", 1);
```

Comment:
These should be in properties file.  Also consider if case insensitive string comparisons might be appropriate.